core/src/main/scala/anorm/SqlRequestError.scala

```diff
@@ -36,7 +36,10 @@ object ColumnNotFound {
     ColumnNotFound(column, row.metaData.availableColumns)
 }
 
-case class UnexpectedNullableFound(message: String) extends SqlRequestError
+case class UnexpectedNullableFound(reason: String) extends SqlRequestError {
```

Comment:
Was going to backport but I think we probably don't want to change the this API if we did that, right?